### PR TITLE
Feature/updated readme

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

**Comment by ralphbean** [comment]
This should also fix #8.

File: rube.core/README.rst [hunk 1]
```diff
@@ -1,86 +1,92 @@
...
+- ``@rube.core.collect_har`` will collect HAR performance data on your
+  websites.  You have to do a little extra work (including setting up
+  browsermob-proxy) in order to get this work.  See below.
+
+----
+
+Running the tests will open up Firefox in X which is a bit of a pain
+sometimes.  If you want, you can run the tests in headless mode by setting
+``headless=1`` in setup.cfg.  Doing so will require that you have
+``xorg-x11-server-Xvfb`` installed via yum, however.
```
Comment (pypingou) [inline rube.core/README.rst:75]: On Fedora, is rube.core generic enough to be used on other systems?

File: rube.fedora/README.rst [hunk 2]
```diff
@@ -1 +1,114 @@
...
+    | '__| | | | '_ \ / _ \
+    | |  | |_| | |_) |  __/
+    |_|   \__,_|_.__/ \___|.fedora
+
+Automated browser tests for `Fedora Infrastructure
+<http://fedoraproject.org/wiki/Infrastructure>`_ with
+`selenium <http://docs.seleniumhq.org/>`_.
+
+It tests against the **staging** environment and attempts to cover
+most of the apps listed at `apps.fp.o <https://apps.fedoraproject.org>`_.
```
Comment (pypingou) [inline rube.fedora/README.rst:19]: apps.fp.o vs apps.fedoraproject.org ?

File: rube.fedora/README.rst [hunk 3]
```diff
@@ -1 +1,114 @@
...
+
+Running
+-------
+
+::
+
+    $ sudo yum install -y python-virtualenvwrapper gcc-g++
```
Comment (pypingou) [inline rube.fedora/README.rst]: I'm always against yum install -y :)

File: rube.fedora/README.rst [hunk 4]
```diff
@@ -1 +1,114 @@
...
+- https://docs.stg.fedoraproject.org/en-US/index.html
+- https://apps.stg.fedoraproject.org/busmon/
+- https://stg.fedoraproject.org/
+- https://apps.stg.fedoraproject.org/packages/
+
+No Staging
+----------
+
+Here is a list of all the sites that do not have a staging equivalent from
+`apps.fp.o https://apps.fedoraproject.org`_.
```
Comment (pypingou) [inline rube.fedora/README.rst]: same question here regarding apps.fp.o

Are these sites tested in rube.fedora? It isn't clear to me from this description

**Comment by pypingou** [comment]
Few comments but minor points. :+1: for me

File: rube.core/README.rst (hunk 1 above)
Comment (ralphbean) [inline rube.core/README.rst:75]: It is!  @gregjurman uses `rube.core` for completely different tests at his workplace.

File: rube.fedora/README.rst (hunk 2 above)
Comment (ralphbean) [inline rube.fedora/README.rst:19]: It will looks nice when its rendered, but still be clickable?

File: rube.fedora/README.rst (hunk 3 above)
Comment (ralphbean) [inline rube.fedora/README.rst]: Oh!  good call.

File: rube.fedora/README.rst (hunk 4 above)
Comment (ralphbean) [inline rube.fedora/README.rst]: You're right, that's not clear.  I'll add a note there.

**Comment by ralphbean** [comment]
Thanks @pypingou!

File: rube.core/README.rst (hunk 1 above)
Comment (pypingou) [inline rube.core/README.rst:75]: Then maybe mention that it's the name of this package on Fedora (might vary on other OS)

**Pull request merged**